### DISCUSSION TITLE
ERD Link for Has-many relation [Jun 07]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5898] [Discord Threads]
> 🙋‍♂️ **davidcafor**  
#### Is this possible to modify?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1104445294873170010/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Not possible to modify. These are the virtual columns that NocoDB creates to display linked record information.  
#  
> 🙋‍♂️ **davidcafor**  
#### so it's doing it right?  
#  
> 🙋‍♂️ **davidcafor**  
#### in mySql Inventario only have id, idProducto (FK productos id), idTienda (FK tiendas id) , cantidad, ultima actualizacion  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Yes. Both "Invenatio L..." and "Productos" are virtual columns managed by NocoDB based on relationships identified within the table. In this case, it's has-many. "Inventario L..." has many "Productos".  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1104445295116427395)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
